### PR TITLE
Mlucassen/traveling salesman

### PR DESCRIPTION
**Pull request by maximilianluc**
Hi! 

This is the pull request for the traveling salesman QIO demo.

Let me know if changes are needed!

### CODE REVIEW

**Comment by anraman** [comment]
Hi @maximilianluc thanks for creating this! I will review probably early next week. I will probably create a PR into your fork if there are large changes and request your review on that before we approve this main PR.

**Comment by maximilianluc** [comment]
Hi @anraman, perfect! thank you.

File: samples/traveling-salesman/README.md
Comment (anraman) [inline samples/traveling-salesman/README.md:31]: Please update these points to accurately reflect what points are covered in the sample

Comment (anraman) [reply samples/traveling-salesman/README.md:31]: (see the other samples for examples)

File: samples/traveling-salesman/README.md
Comment (anraman) [inline samples/traveling-salesman/README.md:42]:
```suggestion
There are three ways to run the sample:
```

File: samples/traveling-salesman/README.md
Comment (anraman) [inline samples/traveling-salesman/README.md:46]: Would recommend adding some instructions on what to do with the HTML page and perhaps a screenshot of the end result, so users know what to expect

File: samples/traveling-salesman/README.md
Comment (anraman) [inline samples/traveling-salesman/README.md:69]:
```suggestion
- **[traveling-salesman.ipynb](https://github.com/microsoft/qio-samples/blob/main/samples/traveling-salesman/traveling-salesman.ipynb)**: Jupyter Notebook version of this sample.
- **[traveling-salesman.py](https://github.com/microsoft/qio-samples/blob/main/samples/traveling-salesman/traveling-salesman.py)**: Standalone Python version of this sample.
- **[traveling-salesman.html](https://github.com/microsoft/qio-samples/blob/main/samples/traveling-salesman/traveling-salesman.html)**: HTML version of this sample.
```

**Review by anraman — CHANGES_REQUESTED** [review]
First pass at a review, going to go through the notebook/python/HTML docs separately and PR those changes into your branch @maximilianluc but figured you could get these changes checked off while I do that :)

**Comment by anraman** [comment]
As I review the sample I will be making it gender-neutral to align with our inclusivity guidelines - just a heads-up that you'll see lots of wording changes to that effect!

**Comment by maximilianluc** [comment]
Haha I didn't see these were outdated - so I did update the readme. (html example picture and more concise learning goals).  Added quantum monte carlo and tabu to the imports to the code(s).

File: samples/traveling-salesperson/traveling-salesperson.py
Comment (anraman) [inline samples/traveling-salesperson/traveling-salesperson.py:16]:
```suggestion

```

Comment (anraman) [reply samples/traveling-salesperson/traveling-salesperson.py:16]: Forgot to remove this line?

**Review by anraman — CHANGES_REQUESTED** [review]
@maximilianluc thanks for all your effort here! I'll approve once you address the last comment and then we will wait for @frtibble to review before merging :)

**Comment by maximilianluc** [comment]
Awesome! thanks for your help @anraman!

**Review by anraman — APPROVED** [review]
Great job, thanks for submitting!

**Pull request merged**
